src/backends/onnxruntime.c

```diff
@@ -270,6 +270,12 @@ RAI_Model *RAI_ModelCreateORT(RAI_Backend backend, RAI_Device device, int64_t de
   if (device == RAI_DEVICE_GPU) {
     OrtSessionOptionsAppendExecutionProvider_CUDA(session_options, deviceid);
   }
+#else
```

Comment:
`RAI_ONNXRUNTIME_USE_CUDA` doesn't seem to propagate with `-DDEVICE=gpu` in cmake

Reply:
Rearranged the CMAKE file. It doesn't seem to break anything else. Please let me know

Reply:
Changed the order in CMAKE file. It should not break anything else but appreciate if you could take a look